src/main/kotlin/team/themoment/gsmNetworking/common/util/UUIDUtils.kt

```diff
@@ -0,0 +1,94 @@
+package team.themoment.gsmNetworking.common.util
+
+import java.time.*
+import java.util.*
+
+class UUIDUtils {
+    companion object {
+        private const val MAX_EPOCH_MILLIS = 95649119999000L // 5000-12-31 23:59:59 GMT
+        private const val MIN_EPOCH_MILLIS = 0L // 1970-1-1 00:00:00 GMT
+
+        val MAX_TIME: Instant = Instant.ofEpochMilli(MAX_EPOCH_MILLIS)
+        val MIN_TIME: Instant = Instant.ofEpochMilli(MIN_EPOCH_MILLIS)
+
+        /**
+         * 주어진 epoch 밀리초를 사용하여 가장 작은 v7 UUID를 생성합니다.
+         *
+         * @param epochMilli 특정 시간을 나타내는 epoch 밀리초 값입니다.
+         * @return epoch 밀리초에서 생성된 가장 작은 v7 UUID입니다.
+         */
+        private fun generateSmallestUUIDv7(epochMilli: Long): UUID {
+            // 지정된 위치에 '-'를 삽입하여 포맷된 UUID 문자열을 생성합니다.
+            // 결과적으로 형식은 "xxxxxxxx-xxxx-7000-0000-000000000000"와 같습니다.
+            val hex = String.format("%012x-7000-0000-000000000000", epochMilli)
+                .replaceRange(8, 8, "-")
+            return UUID.fromString(hex)
+        }
+
+        /**
+         * 가장 작은 v7 UUID를 생성합니다.
+         *
+         * @param time 변환할 LocalDateTime 객체
+         * @return epoch 밀리초 값
+         */
+        fun generateSmallestUUIDv7(time: LocalDateTime): UUID {
+            val epochMilli = getEpochMilli(time)
+            return generateSmallestUUIDv7(epochMilli)
+        }
+
+        /**
+         * 가장 작은 v7 UUID를 생성합니다.
+         *
+         * @param time 변환할 Instant 객체
+         * @return epoch 밀리초 값
+         */
+        fun generateSmallestUUIDv7(instant: Instant): UUID {
+            val epochMilli = instant.toEpochMilli()
+            return generateSmallestUUIDv7(epochMilli)
+        }
+
+        /**
+         * 주어진 UUID에서 시잔 정보를 반환합니다.
```

Comment:
코드 읽다가 오타 하나 발견했습니다.
```suggestion
         * 주어진 UUID에서 시간 정보를 반환합니다.
```

Reply:
[1b6c393](https://github.com/themoment-team/GSM-Networking-server/pull/17/commits/1b6c3931d6f230255580c2f5bee96bdbe4e058cd) 수정하였습니다.